samples/gen-go/server/router.go

```diff
@@ -50,18 +52,35 @@ func (s *Server) Serve() error {
 		s.l.Info("please provide a kvconfig.yml file to enable app log routing")
 	}
 
-	if lightstepToken := os.Getenv("LIGHTSTEP_ACCESS_TOKEN"); lightstepToken != "" {
-		tags := make(map[string]interface{})
-		tags[lightstep.ComponentNameKey] = "swagger-test"
-		lightstepTracer := lightstep.NewTracer(lightstep.Options{
-			AccessToken: lightstepToken,
-			Tags:        tags,
-			UseGRPC:     true,
-		})
-		defer lightstep.FlushLightStepTracer(lightstepTracer)
-		opentracing.InitGlobalTracer(lightstepTracer)
+	if signalfxToken := os.Getenv("SIGNALFX_ACCESS_TOKEN"); signalfxToken != "" {
+		ingestUrl := os.Getenv("SIGNALFX_INGEST_URL")
+		if ingestUrl == "" {
+			ingestUrl = "https://ingest.signalfx.com"
+		}
+
+		// Create a Jaeger HTTP Thrift transport
+		transport := transport.NewHTTPTransport(ingestUrl+"/v1/trace",
+			transport.HTTPBasicAuth("auth", signalfxToken))
+
+		cfg, err := jaegercfg.FromEnv()
+		if err != nil {
+			log.Fatal("Could not parse Jaeger env vars: ", err.Error())
+		}
+		cfg.ServiceName = "swagger-test"
+		cfg.Sampler = &jaegercfg.SamplerConfig{
+			Type:  jaeger.SamplerTypeRateLimiting,
+			Param: 500,
+		}
+
+		signalfxTracer, closer, err := cfg.NewTracer(jaegercfg.Reporter(jaeger.NewRemoteReporter(transport)))
+		if err != nil {
+			log.Fatal("Could not initialize jaeger tracer: ", err.Error())
+		}
+		defer closer.Close()
+
+		opentracing.SetGlobalTracer(signalfxTracer)
```

Comment:
Is there a way to add a default set of tags for traces? If so, consider checking these env variables that we add to every deploy:
_APP_NAME
_BUILD_ID
_DEPLOY_ENV
_TEAM_OWNER